src/Tribe/Editor.php

```diff
@@ -138,7 +139,17 @@ public function update_post_content_to_blocks( $post ) {
 
 		foreach ( $blocks as $key => $block_param ) {
 			$slug = reset( $block_param );
-			$params = end( $block_param );
+			/**
+			 * Add an opportunity to set the default params of a block when migrating from classic into
+			 * blocks editor.
+			 *
+			 * @since TBD
+			 *
+			 * @param mixed $params Either array if set to values or slug string
+			 * @param string $slug Name of the block edited
+			 * @param WP_Post $post Post that is being affected
+			 */
+			$params = apply_filters( 'tribe_blocks_editor_update_classic_content_params', end( $block_param ), $slug, $post );
```

Comment:
Why aren't we sending `$block_param` as the first param and using `end` instead?

Reply:
Great question, `end` is from the initial code here, however it has a good reason why as the `$block_params` is something as the following shape.

```php
// "regular" block
array( 'block/slug', array(
	'extra' => 'params'
	'over' 	=> 'here,
));

// no params
// "regular" block
array( 'block/slug' );
```

So `end( $params )` is either an array of params or the `slug` of the block.

Reply:
Cool, thanks for the clarification! Looking great